microshift_running_apps/microshift-applications.adoc

```diff
@@ -12,4 +12,9 @@ include::modules/microshift-manifests-overview.adoc[leveloffset=+1]
 
 include::modules/microshift-manifests-override-paths.adoc[leveloffset=+1]
 
-include::modules/microshift-applying-manifests-example.adoc[leveloffset=+1]
\ No newline at end of file
+//[id="additional-resources_applications-with-microshift_{context}"]
```

Comment:
Assembly level additional resources sections need an ID, is there a reason this one is commented out? If not, this should be uncommented (and can probably be shortened to keep it simple, but that part is optional, except for removing that first underscore):
```suggestion
[id="additional-resources_{context}"]
```

Reply:
done.